code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.cs

```diff
@@ -0,0 +1,29 @@
+using System; 
+
+class LongestRepeatingSubsequence 
+{ 
+    static int longestRepeatingSubsequence(string str) 
```

Comment:
Should be upper camel case for C#.

Reply:
Class name is in upper case. Function name and class name cannot be same.